R/test-that.R

```diff
@@ -58,7 +58,7 @@ test_code <- function(test, code, env = test_env(), skip_on_empty = TRUE) {
       e$end_frame <- sys.nframe() - debug_end - 1L
     }
 
-    e$test <- test %||% "(unknown)"
+    e$test <- test %||% "(code run outside of a test)"
```

Comment:
```suggestion
    e$test <- test %||% "(code run outside of `test_that()`)"
```
Maybe? "outside of a test" might be taken differently, as everything in `test-foo.R` might be considered "inside of a test".